pom.xml

```diff
@@ -105,7 +105,7 @@
         <hadoop-aws.version>3.1.4</hadoop-aws.version>
         <aws-java-sdk-bundle.version>1.11.271</aws-java-sdk-bundle.version>
         <spotless.version>2.29.0</spotless.version>
-        <logback.version>1.5.7</logback.version>
+        <logback.version>1.2.3</logback.version>
```

Comment:
Why we should downgrade logback version?

Reply:
1. Despite upgrading to version 1.5.7, there remains an unresolved Common Vulnerabilities and Exposures (CVE) issue in this version.
2. Upon upgrading to 1.5.7, the log4j-slf4j-impl jar file is absent from the project, resulting in failed logging attempts.

So reverted as of now. Will check more on this later